### PR TITLE
add an error remind in dpgen_fp

### PR DESCRIPTION
**Pull request by goodluck-xyz**
Describe in detail what's wrong with the INCAR in the FADs https://github.com/deepmodeling/dpgen/discussions/898.
The accurate error reason help the user solve the problem as soon as possible.

Signed-off-by: Yunfan Xu <54764135+goodluck-xyz@users.noreply.github.com>

### CODE REVIEW

**Comment by HuangJiameng** [comment]
Please set a proper title to briefly describe your PR.

**Comment by njzjz** [comment]
Please resolve conflicts.

File: dpgen/generator/run.py [hunk 1]
```diff
@@ -3243,6 +3243,9 @@ def post_fp_vasp (iter_index,
                         assert(use_ele_temp)
                         ele_temp = job_data['ele_temp']
                         all_te.append(ele_temp)
+            elif len(_sys) >= 2:
+                raise RuntimeError("The vasp parameter NSW should be set as 1") 
+                icount+=1
```
Comment (njzjz) [inline dpgen/generator/run.py:3248]: Line 3248 will not be executed when line 3247 raises an error. So I don't know the role of line 3248.

Comment (goodluck-xyz) [reply dpgen/generator/run.py:3248]: I misunderstand the function RuntimeError(). The line 3248 should be removed.

**Review by njzjz — APPROVED** [review]

**Pull request merged**